<?xml version="1.0"?>
<launch>    
    <node pkg="tf" type="static_transform_publisher" name="tf_base_link_to_torso" args="0 0 0   0 0 0 1 base_link torso 100" /> 
    <node pkg="tf" type="static_transform_publisher" name="tf_head_to_cam" args="0 0 0.1  0.0 0.076 0.0 0.997 Head camera_rgb_frame 100" />  
    
    <include file="$(find gem)/launch/gem_ros.launch" />
    
    <node name="ics_fusion" pkg="ics_fusion" type="ics_fusion_node"  output="screen" >
        <rosparam file="$(find ics_fusion)/config/slam_params.yaml" command="load"/> 
        <param name="publish_volume" type="bool" value="true" />
        <param name="publish_points" type="bool" value="false" />
        <param name="publish_points_rate" type="int" value="10" />
        
        <param name="cam_info_topic" type="str" value="/camera/depth_registered/sw_registered/camera_info" />
        <param name="depth_topic"    type="str"    value="/camera/depth_registered/sw_registered/image_rect" />
        <param name="rgb_topic"      type="str" value="/camera/rgb/image_rect_color" />
        
        <param name="cut_off"      type="double" value="0.8" />
        <param name="cut_off"      type="double" value="0.8" />
        
    </node>    
    
<!--     <node pkg="rosbag" type="play" name="rosbag" required="true" args="- - clock /home/tavu/jetson_ws/bags/icraWS1.bag"/> -->
</launch>
